core/src/main/java/com/hcl/ecomm/core/services/impl/MySqlServiceImpl.java

```diff
@@ -0,0 +1,107 @@
+package com.hcl.ecomm.core.services.impl;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.engine.EngineConstants;
+import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+
+import com.day.commons.datasource.poolservice.DataSourcePool;
+import com.hcl.ecomm.core.services.CustomEmailService;
+import com.hcl.ecomm.core.services.MySqlService;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+import java.sql.SQLException;
+import javax.sql.DataSource;
+
+@Component
+public class MySqlServiceImpl implements MySqlService {
+
+	protected final Logger LOG = LoggerFactory.getLogger(this.getClass());
+
+	@Reference
+	private DataSourcePool source;
+	
+	@Reference
+    CustomEmailService customEmailService;
+
+	@Override
+	public void userComplaintSubmission(String first_Name, String last_Name, String email, String subject,
+			String complaint) {
+
+		LOG.info("userComplaintSubmission method start. email="+email);
```

Comment:
@rakeshkumar-verma : pl update log.info to log.debug

Reply:
@rishabh-behl  done